app/UDB2IncomingEventServicesProvider.php

```diff
@@ -194,21 +185,12 @@ function (Application $app) use ($importFromSapi) {
 
                 $enricher->setLogger($app['logger.xml-imports.enricher']);
 
-                if ($importFromSapi) {
-                    $eventUrlFormat = $app['udb2_cdbxml_enricher.event_url_format'];
```

Comment:
Nit: We can also remove these parameters here now: https://github.com/cultuurnet/udb3-silex/blob/cf8d528775aac8574170bcff24c530f6cb1a56d9/bootstrap.php#L1077-L1078

Reply:
Good catch. I pushed it in the next PR to avoid having to merge everything: https://github.com/cultuurnet/udb3-silex/pull/654/commits/d79caafd4dee2b78967a9fb4fe0624f074f240fc

Reply:
Perfect!